docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,148 @@
+| "starts with" expression in any selector                | NO                           |
+| "ends with" expression in any selector                  | NO                           |
+| `spec.PreDNAT`                                          | NO                           |
+| `spec.ApplyOnForward`                                   | NO                           |
+| `spec.DoNotTrack`                                       | NO                           |
+| `spec.Ingress.ICMP` or `spec.Egress.ICMP`               | NO                           |
+| `spec.Ingress.NotICMP` or `spec.Egress.NotICMP`         | NO                           |
+| `spec.Ingress.NotProtocol` or `spec.Egress.NotProtocol` | NO                           |
+| `spec.Ingress.NotPorts` or `spec.Egress.NotPorts`       | NO                           |
+| `spec.Ingress.Metadata` or `spec.Egress.Metadata`       | NO                           |
+| `spec.Ingress.NotSelector` or `spec.Egress.NotSelector` | NO                           |
+| `spec.Ingress.HTTP` or `spec.Egress.HTTP`               | NO                           |
+| `spec.Ingress.NotNets` or `spec.Egress.NotNets`         | NO                           |
+| All other features                                      | YES                          |
+
+## Restart Pods
+
+After Antrea is installed in the cluster, the script will restart all Pods in the cluster in-place
+by deploying a DaemonSet named `antrea-migrator`, which will run a Pod on each Node. The `antrea-migrator`
+Pod will kill all Pods' containerd task on each Node, and the containerd task will be restarted
+by the containerd service. In this way, all Pods in the cluster will be restarted in-place and do not
+need to be rescheduled and recreated.
+
+The restart result is as follows, every Pod's `RESTARTS` will be increased by 1:
+
+```bash
+$ kubectl get pod -n migrate-test -o wide
+NAME                               READY   STATUS    RESTARTS      AGE    IP          NODE          NOMINATED NODE   READINESS GATES
+migrate-example-6d6b97f96b-29qbq   1/1     Running   1 (24s ago)   2m5s   10.10.1.3   test-worker   <none>           <none>
+migrate-example-6d6b97f96b-dqx2g   1/1     Running   1 (23s ago)   2m5s   10.10.1.6   test-worker   <none>           <none>
+migrate-example-6d6b97f96b-jpflg   1/1     Running   1 (23s ago)   2m5s   10.10.1.5   test-worker   <none>           <none>
+```
+
+## Uninstall old CNI
+
+After all Pods are restarted, the script will uninstall the old CNI by using `kubectl delete -f <old-cni-yaml>`.
+The old CNI manifest is specified by the `--calico-manifest` or `--flannel-manifest` option. If this option
+is not specified, the script will try to use the latest version.
+
+```bash
+# Uninstall Calico
+./hack/migrate-to-antrea.sh --from calico --calico-manifest /tmp/calico/calico.yaml
+# Uninstall Flannel
+./hack/migrate-to-antrea.sh --from flannel --flannel-manifest /tmp/flannel/flannel.yaml
+```
```

Comment:
This confused me, I suppose there is no need to run this step again for old CNI deletion, right?
I feel you may add a new line before the script like:
"You should run `migrate-to-antrea.sh` as the following examples If you want to uninstall the CNI with a specific manifest instead of the latest one."

Reply:
This example shows different CNI's commands, it does cause some misunderstanding, removed.